ghcide/src/Development/IDE/Types/HscEnvEq.hs

```diff
@@ -58,15 +57,19 @@ updateHscEnvEq oldHscEnvEq newHscEnv = do
   let update newUnique = oldHscEnvEq { envUnique = newUnique, hscEnv = newHscEnv }
   update <$> Unique.newUnique
 
+toAbsolute :: FilePath -> FilePath -> FilePath
```

Comment:
need to find a place to put all the `toAbsolute` functions so that it can be shared